pkg/porter/explain.go

```diff
@@ -114,6 +114,8 @@ func (s SortPrintableAction) Swap(i, j int) {
 	s[i], s[j] = s[j], s[i]
 }
 
+var porterInternalParams = []string{"porter-debug"}
```

Comment:
If we make this a `map[string]struct{}` (optimized for memory) or `map[string]bool` (not optimized but can be easier to understand sometimes), and then checkIsInternalParam can be simplified, where it returns the second return value from querying the map

```golang
var porterInternalParams = map[string]struct{}{
    "porter-debug": struct{}{},
}

// ...

// ok holds if the specified key is present in the map
_, ok := porterInternalParams[inputParam]
return ok
```